ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -32,6 +35,15 @@ import { useI18nContext } from '../../../../../hooks/useI18nContext';
 import useAlerts from '../../../../../hooks/useAlerts';
 import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
 
+export type FrictionModalConfig = {
```

Comment:
Is it cleaner and easier to maintain if we treat this as an alternate component, since we seem to be splitting the component into two states.

It could use the original `AlertModal` internally through generic properties like `children` or `buttons` etc.

Is the term friction modal a little ambiguous since I think of all of the alert modals as friction modals, maybe `ConfirmAlertModal`?

Reply:
Great suggestions, I extracted the friction modal and created a new component `ConfirmAlerModal` as you suggested.